besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -4126,7 +4128,7 @@ private void networkValuesCanBeOverridden(final String network) throws Exception
     final ArgumentCaptor<EthNetworkConfig> networkArg =
         ArgumentCaptor.forClass(EthNetworkConfig.class);
 
-    verify(mockControllerBuilderFactory).fromEthNetworkConfig(networkArg.capture(), any());
+    verify(mockControllerBuilderFactory).fromEthNetworkConfig(networkArg.capture(), any(), any());
```

Comment:
argh, those extra params were quite painful in the end 😢

Reply:
Hopefully someone else doesn't disagree and ask you to make another constructor 😅

Reply:
^ For context, me and @gfukushima discussed this and I preferred making the parameter explicit in order to surface the pain rather than hide it away...I think if and when we add yet another param to fromEthNetworkConfig we should consider refactoring it, maybe a wrapper object around everything required to build the Controller.